services/net/.vscode/tasks.json

```diff
@@ -180,6 +180,30 @@
         "${workspaceFolder}/syndication/TNO.Services.Syndication.csproj"
       ],
       "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "build-transcription",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/transcription/TNO.Services.Transcription.csproj",
+        "/property:GenerateFullPaths=true",
+        "/consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "publish-transcription",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "publish",
+        "${workspaceFolder}/transcription/TNO.Services.Transcription.csproj",
+        "/property:GenerateFullPaths=true",
+        "/consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
```

Comment:
Not sure if it's required, but I think the other tasks have a `watch-*` too

Reply:
Is there any action to take here? As usual, I'm just copying entries and seeing of they work.

Reply:
I don't know if the `watch-*` is required or not.  You'd have to test running it in debug mode to find out.

Reply:
It seems to work fine without it. What is this supposed to do?